### DISCUSSION TITLE
Can't change repository URL

### DISCUSSION THREAD

**Discussion by DuployerCharles** [codemagic-ci-cd/codemagic-docs#1985] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Is it possible to have an email address or something else to communicate faster ? I have 7 applications repository to change. I don't really want to share it here !

Thank you :)

### Expected results

Repository URL is editable

### Actual results

Repository URL is readonly

**Comment by kulakarla** [comment]
Hello @DuployerCharles 

are you a paying customer? If so, you should have a chat widget for our premium support where we could discuss the issue privately.

**Comment by himesh-cm** [comment]
Hi @DuployerCharles,
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
